### PR TITLE
Delete unused media files and Resize profile pictures

### LINKED ISSUE #110: Delete unused media files and Resize profile pictures
- Resize this image when it is uploaded to save space on our web server
- Delete unused media files when a user changed profile

### PR DESCRIPTION
**Pull request by Leeadar**
- Adding django-cleanup app
- Overwrite old user images when they are updated
- Creating save method in models.py to resize image when it is uploaded to save space on our web server

closes #110

### CODE REVIEW

**Review by nadavspinco — APPROVED** [review]

File: picATrip/settings.py [hunk 1]
```diff
@@ -41,6 +41,7 @@
     'pickATrip_django_apps.apps.pickATrip_django_appsConfig',
     'users.apps.UsersConfig',
     'crispy_forms',
+    'django_cleanup',
```
Comment (maayanhd) [inline picATrip/settings.py:44]: nice addition!

File: users/models.py [hunk 2]
```diff
@@ -9,3 +10,13 @@ class Profile(models.Model):
 
     def __str__(self):
         return f'{self.user.username} Profile'
+
+    def save(self, *args, **kwargs):
+        super().save(*args, **kwargs)
+
+        img = Image.open(self.image.path)
+
+        if img.height > 300 or img.width > 300:
```
Comment (maayanhd) [inline users/models.py:19]: what about testing this change on images saved to the database?

Comment (Yarboa) [reply users/models.py:19]: +1 on that

Comment (Leeadar) [reply users/models.py:19]: I added a test to the image size

**Review by maayanhd — APPROVED** [review]

**Review by Shoval-Ovadia — APPROVED** [review]
LGTM 👍

**Pull request merged**